flexmeasures/data/scripts/cli_tasks/data_delete.py

```diff
@@ -19,6 +19,29 @@ def fm_delete_data():
     """FlexMeasures: Delete data."""
 
 
+@fm_delete_data.command("account-role")
+@with_appcontext
+@click.option("--name", required=True)
+def delete_account_role(name: str):
+    """
+    Delete an account role, if it has no users, otherwise say which ones.
```

Comment:
"users" should be "accounts". And if the role has accounts attached to it, the role is detached from those accounts and **then the role is still deleted**. The latter isn't clear from this sentence.